src/components/form-elements/Select.jsx

```diff
@@ -103,6 +122,14 @@ const Select = (props: SelectPropsType) => {
     };
   }
 
+  if (options.length && groupedOptions.length) {
+    throw {
+      name: 'WrongOptions',
+      message:
+        'Select: should passed be either options or groupedOptions prop!',
+    };
+  }
```

Comment:
Maybe we can use `invariant` function here, @coderitual wdyt?

Reply:
yes, exactly

Reply:
@ensisf you can check the example usage of this function in `Accordion` component.